Service/Xml.php

```diff
@@ -141,7 +141,7 @@ private function exportObject(DataObject $object, $useRecursion=true, $addFields
         if ($this->includeVariants) {
             $children = $object->getChildren([DataObject\AbstractObject::OBJECT_TYPE_VARIANT]);
             foreach ($children as $child) {
-                $childData =  $this->exportObject($child);
+                $childData =  $this->exportObject($child, true, !$this->omitRelationObjectFields);
```

Comment:
As omitRelationObjectFields is available to all methods, do we really need to pass this as a parameter?

Reply:
I have oriented myself on the existing implementations, like as for the method getForTypeManyToManyObjectRelation.
Without these parameters, this place would currently not consider this flag in Children and I could observe this behavior in the API.